src/Datadog.Trace.ClrProfiler.Managed/Integrations/AdoNet/SqlTags.cs

```diff
@@ -17,7 +18,7 @@ internal class SqlTags : InstrumentationTags
 
         public string DbType { get; set; }
 
-        public string InstrumentationName => AdoNetConstants.IntegrationName;
+        public string InstrumentationName => IntegrationRegistry.GetName(AdoNetConstants.IntegrationId);
```

Comment:
How much would we save if we initialized a static field with `IntegrationRegistry.GetName(AdoNetConstants.IntegrationId)` and then just referenced that? This would also apply to the other InstrumentationTags as well

Reply:
I don't think it'll save much, but the change is trivial so I don't see any reason not to do it :p